_posts/2022-11-18-sc-candidates.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: 2022 Steering Committee Candidate Information
+tags: [election]
+posted_by: Christina Maimone
+---
+
+Candidates for the [2022 US-RSE Steering Committee
+election](https://us-rse.org/about/election/) are listed below.  Four of these
+candidates will be elected for a two-year term starting January 2023.
+
+If you have questions for the candidates, or questions about the election,
+please use the #election channel on Slack. You must be [registered as a member](https://us-rse.org/join)
+by Friday, November 20th to be eligible to vote in this election.
```

Comment:
Is it Friday Nov 18 or Sunday Nov 20th?

Reply:
Nov 20th - I got the day of the week wrong